<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setup</class>
 <widget class="QWidget" name="Setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">gridline-color: rgb(0, 0, 0);
background-color: rgb(70, 70, 70);
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Hostname</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hostBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
gridline-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
gridline-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="secretCheck">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgba(70, 70, 70, 70);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Secret Password</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="configBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Default Gateway</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipBox1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
gridline-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipBox2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
gridline-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipBox3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
gridline-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipBox4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
gridline-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Banner MOTD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="motdBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
gridline-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
gridline-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Banner Login</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="domainCheck">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgba(70, 70, 70, 70);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Ip Domain Lookup</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="encryptCheck">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgba(70, 70, 70, 70);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Encrypt Password</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>hostBox</tabstop>
  <tabstop>domainCheck</tabstop>
  <tabstop>pwBox</tabstop>
  <tabstop>secretCheck</tabstop>
  <tabstop>ipBox1</tabstop>
  <tabstop>ipBox2</tabstop>
  <tabstop>ipBox3</tabstop>
  <tabstop>ipBox4</tabstop>
  <tabstop>motdBox</tabstop>
  <tabstop>loginBox</tabstop>
  <tabstop>clearBtn</tabstop>
  <tabstop>configBtn</tabstop>
  <tabstop>exitBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
